DOKS-automatic-node-repair/README.md

```diff
@@ -1,202 +1,172 @@
 # Replacing Unhealthy DOKS Cluster Nodes Automatically
 
-## Problem Description
+This tutorial provides an automated way to recycle unhealthy nodes in a DigitalOcean Kubernetes (DOKS) cluster.  When a node in a DOKS cluster is unhealthy, it is a manual and cumbersome process to replace the node. Without replacing the nodes, the cluster will operate at lower capacity because the unhealthy nodes will not run any Pods. 
 
-When a node in a `DOKS` cluster is unhealthy, it is a manual, cumbersome process to replace the node automatically. Without doing it, cluster will operate at lower capacity, because the unhealthy nodes will not run any Pods.
+You can automatically replace unhealthy nodes using one of the following applications:
 
-This tutorial provides an automated way to `recycle` the `unhealthy nodes` in `DOKS`. 
-
-
-## Solution:
-
-We can achieve this using one of the following:
  - [Digital Mobius](https://github.com/Qovery/digital-mobius)
+ 
  - [Draino](https://github.com/planetlabs/draino) and [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
 
-For the use case described in this tutorial, we will use [Digital Mobius](https://github.com/Qovery/digital-mobius). We will deploy to a running DOKS cluster and observe how well it handles cluster nodes recycling.
-
-Main reasons to consider `Digital Mobius` are:
- - Specifically built for `DOKS` cluster nodes recycling.
- - Open source (written in [Go](https://golang.org)).
- - Simplicity and ease of configuration.
- - [Helm chart](https://github.com/Qovery/digital-mobius/tree/main/charts/Digital-Mobius) ready and available for easy Kubernetes deployment (or [artifacthub.io](https://artifacthub.io/packages/helm/digital-mobius/digital-mobius))
-
-To learn more, see [Overview](#Overview).
-
-## Set Up DOKS Cluster
-
-### Requirements
+For this tutorial, we will deploy Digital Mobius on a running DOKS cluster. Digital Mobius is an open-source application written in Go and is specifically built for DOKS cluster nodes recycling. The application monitors DOKS cluster nodes that are in an unhealthy state at specified regular interval. A node is considered to be unhealthy if the [node condition](https://kubernetes.io/docs/concepts/architecture/nodes/#condition) is `Ready` and the status is `False` or `Unknown`. The application recreates the affected node(s) using the DigitalOcean [Delete Kubernetes Node API](https://docs.digitalocean.com/reference/api/api-reference/#operation/delete_kubernetes_node). It is [Helm chart-ready](https://github.com/Qovery/digital-mobius/tree/main/charts/Digital-Mobius) and available for easy Kubernetes deployment (or [artifacthub.io](https://artifacthub.io/packages/helm/digital-mobius/digital-mobius)).
```

Comment:
It will be nice to add that "Digital Mobius" is contributed by Qovery. It is also obvious to the users from the GitHub page, so I leave it to you.

Reply:
Added